pkgs/os-specific/linux/sysklogd/fix-includes.patch

```diff
@@ -0,0 +1,118 @@
+diff -u sysklogd-1.5.1.orig/klogd.c sysklogd-1.5.1/klogd.c
```

Comment:
please describe that the patch is required for musl build only.
Otherwise, someone could try to build without it (on glibc) and remove it on next update, thinking it is not required anymore.

Reply:
added a comment at the top of the patch file as well as changed its name, PTAL

yeah breaking things for musl seems like it might be unavoidable without some kind of testing, but don't know nix enough yet to have any suggestions (other than tracking some packages in my own hydra)